_rules/image-button-has-name-59796f.md

```diff
@@ -53,31 +57,31 @@ There is a known combination of a popular browser and assistive technology that
 Image button element with accessible name through `alt` attribute
 
 ```html
-<input type="image" name="submit" src="button.gif" alt="Submit" />
+<input type="image" name="submit" src="button.gif" alt="Submit" value="Submit"/>
```

Comment:
:-1: to adding value attributes here. They're not required. Just turn off the rule that checks for them here.